This test does not make sense since the file is not yet installed.

Index: console_bridge-1.0.1/test/CMakeLists.txt
===================================================================
--- console_bridge-1.0.1.orig/test/CMakeLists.txt
+++ console_bridge-1.0.1/test/CMakeLists.txt
@@ -77,7 +77,7 @@ if(WGET_EXE)
                    "--counting=detailed"
                    "--extensions=cpp,h"
                    "--linelength=100"
-                   "--filter=-build/c++11,-runtime/references,-whitespace/braces,-whitespace/indent,-whitespace/parens,-whitespace/semicolon"
+                   "--filter=-build/c++11,-runtime/references,-whitespace/braces,-whitespace/indent,-whitespace/parens,-whitespace/semicolon,-build/header_guard"
            "${PROJECT_SOURCE_DIR}/include/console_bridge/console.h"
            "${PROJECT_SOURCE_DIR}/src/console.cpp"
   )
